### PR TITLE
Feature/line profile

### PR DESCRIPTION
**Pull request by timbeccue**
This PR adds code to connect the frontend with the quickanalysis server ([story](https://www.pivotaltracker.com/story/show/175941579)). 

The primary addition is the component used to start the analysis and display the results: `/src/components/LineProfileInspection.vue`.

### CODE REVIEW

File: src/components/ImageView.vue [hunk 1]
```diff
@@ -82,6 +82,7 @@
           </b-field>
         </b-field>
 
+        <b-field>
```
Comment (jnation3406) [inline src/components/ImageView.vue:85]: indentation for this section

File: src/components/LineProfileInspection.vue [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+      <button 
+        title="first draw a line on the image to inspect"
+        @click="getLineProfile"
+        class="button" 
+        :class="{'is-loading': analysisInProgress}">
+        get line profile
+      </button>
+    </b-field>
+
+    <!--div id="line-inspection-chart" /-->
```
Comment (jnation3406) [inline src/components/LineProfileInspection.vue:24]: is this a comment or commented out code?

File: src/store/modules/dev.js [hunk 3]
```diff
@@ -18,6 +18,8 @@ const state = {
 
     logs_ws_endpoint: "wss://l932c9n1r3.execute-api.us-east-1.amazonaws.com/prod",  // prod
     //logs_ws_endpoint: "wss://8t7v3l1ok5.execute-api.us-east-1.amazonaws.com/dev", // dev
+
+    quickanalysis_endpoint: "http://quickanalysis.photonranch.org",
```
Comment (jnation3406) [inline src/store/modules/dev.js:22]: you'll probably want all these URLs to be passed in eventually and set by env variable or something in your deploy, so you can support dev deployments hitting dev versions of services eventually

**Review by jnation3406 — APPROVED** [review]

File: src/store/modules/dev.js (hunk 3 above)
Comment (timbeccue) [reply src/store/modules/dev.js:22]: Good idea. I'll keep this in mind.

**Pull request merged**